lib/run/src/Obelisk/Run.hs

```diff
@@ -95,7 +95,7 @@ runServeAsset :: FilePath -> [Text] -> Snap ()
 runServeAsset rootPath = Snap.serveAsset "" rootPath . T.unpack . T.intercalate "/"
 
 getConfigRoute :: Map Text Text -> IO (Maybe URI)
-getConfigRoute configs = case Map.lookup "common/route" configs of
+getConfigRoute configs = case Map.lookup "config/common/route" configs of
```

Comment:
this change is interesting -- somehow the configs being passed here include stuff outside the config directory now?

Reply:
No, this was just a poorly-conceived change

Reply:
(the one that made this change necessary, i mean)